src/runtime/int.cpp

```diff
@@ -320,8 +320,8 @@ extern "C" Box* div_i64_i64(i64 lhs, i64 rhs) {
 #if PYSTON_INT_MIN < -PYSTON_INT_MAX
     static_assert(PYSTON_INT_MIN == -PYSTON_INT_MAX - 1, "");
 
-    if (lhs == PYSTON_INT_MIN && rhs == -1) {
-        return longDiv(boxLong(lhs), boxLong(rhs));
+    if (lhs == PYSTON_INT_MIN) {
```

Comment:
I think this fix isn't quite right -- it fixes the cpython test but because they only test against `lhs == INT_MIN`.  We have the same overflow problem with other cases, such as `-sys.maxint / 4`.  I think we should leave the `INT_MIN / -1` case as a special case (since it extra-special), but we'll need some other fix for when `lhs - rhs` overflows.